### PR TITLE
Inquartata trait

### PR DESCRIPTION
**Pull request by Apocarypse**
**_I affirm:_**
- [x] that I agree to Project Topaz's [Limited Contributor License Agreement](http://project-topaz.com/blob/release/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I've _tested my code_ since the last commit in the PR, and will test after any later commits

Added mod enum for Inquartata in modifier.h
Added entries for Inquartata in traits.sql
Applied Inquartata bonus logic to GetParryRate()

### CODE REVIEW

**Review by Kreidos — APPROVED** [review]
Looks good to me!

File: src/map/utils/battleutils.cpp [hunk 1]
```diff
@@ -1723,6 +1723,15 @@ namespace battleutils
                     parryRate = parryRate + issekiganBonus;
                 }
 
+                // Inquartata grants a flat parry rate bonus.
+                if (CCharEntity* PChar = dynamic_cast<CCharEntity*>(PDefender))
```
Comment (ibm2431) [inline src/map/utils/battleutils.cpp]: Sverdheid and Zurko-Bazurko are RUN mobs that players need to fight.

Comment (Kreidos) [reply src/map/utils/battleutils.cpp]: I think based on what was said on the discord, that ibm's idea was that if you aren't needing the trait check, then there's no need to cast to CCharEntity*. You can just use PDefender directly as CBattleEntity has the getMod method. :)

**Review by ibm2431 — APPROVED** [review]

**Review by cocosolos — APPROVED** [review]

**Comment by TeoTwawki** [comment]
@Apocarypse @cocosolos @ibm2431 modifiers in modifier.h and statsu.lua should mirror each other until such time as lua can automagically use the core enums, even if the mod is never used in script so that future editor don't mistakenly use the same ID - this has happened multiple times in the older project history until I became its watchdog in dsp.

**Comment by ibm2431** [comment]
_sees this merged PR in notifications_
_sees Teo as a participant_

"I _did_ miss one, didn't I?"

_clicks_

"Yeeeep. Whoops."

**Pull request merged**